tools/functional-tester/etcd-runner/command/lock_racer_command.go

```diff
@@ -26,18 +26,24 @@ import (
 // NewLockRacerCommand returns the cobra command for "lock-racer runner".
 func NewLockRacerCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "lock-racer",
+		Use:   "lock-racer [name of lock (defaults to 'racers')]",
 		Short: "Performs lock race operation",
 		Run:   runRacerFunc,
 	}
-	cmd.Flags().IntVar(&rounds, "rounds", 100, "number of rounds to run")
+	cmd.Flags().IntVar(&rounds, "rounds", 100, "number of rounds to run. set rounds <= 0 to run forever")
 	cmd.Flags().IntVar(&totalClientConnections, "total-client-connections", 10, "total number of client connections")
 	return cmd
 }
 
 func runRacerFunc(cmd *cobra.Command, args []string) {
-	if len(args) > 0 {
-		ExitWithError(ExitBadArgs, errors.New("lock-racer does not take any argument"))
+	racers := "racers"
```

Comment:
move this directly above `len(args) == 1`; otherwise it's splitting up the `racers` setting logic for no reason